contrib/net/http/roundtripper.go

```diff
@@ -58,8 +59,8 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		span.SetTag(ext.Error, err)
 	} else {
 		span.SetTag(ext.HTTPCode, strconv.Itoa(res.StatusCode))
-		// treat 5XX as errors
-		if res.StatusCode/100 == 5 {
+		// check for 5XX server codes
```

Comment:
I think we can leave out "5XX" since we're not checking for 5XX, but rather whatever is configured.